docs/server/settings.rst

```diff
@@ -273,6 +273,14 @@ Configuration settings for applications used by Pootle.
   Enable spam prevention through a captcha.
 
 
+.. setting:: POOTLE_REPORTS_MARK_FUNC
+
+``POOTLE_REPORTS_MARK_FUNC``
+  Default: ''
+
+  Provide mark data for user daily activity charts in admin Pootle reports
```

Comment:
Something like this might work. Please double-check it, improve it, and test its rendered output.
```
The timeline graph of user's activity within reports can be `optionally marked <https://code.google.com/p/flot-marks/>`_ using this function.

The function receives the ``user`` (``User`` model), and the ``start`` and ``end`` points (timestamps as strings in ``%Y-%m-%d`` format) as parameters. It must return a dictionary of key-value pairs, with
  * `position`, specifying the point in the x-axis where the mark should be set (timestamp), and
  * (optional) `label` specifying the text that will be displayed next to the mark.
```